topo/topo.go

```diff
@@ -543,6 +543,9 @@ func (m *Manager) push(ctx context.Context) error {
 		ns := &corev1.Namespace{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: m.topo.Name,
+				Labels: map[string]string{
+					"admission-webhook": "enabled",
```

Comment:
Are you suggesting an extra label here?

Reply:
i was thinking instead of the "admission-webhook" label. You can operate on all the topologies namespaces but doing 
```
namespaceSelector:
    matchExpressions:
      - key: kne-topology
        operator: Exists
```

Reply:
Ah ok - got it. Thanks makes sense, I'll change this to what you are suggesting.